### PR TITLE
Remove tac command usage which require coreutils

### PR DESCRIPTION
**Pull request by yussufsh**
1. Replace tac command with tail -f
2. Added .gitignore

### CODE REVIEW

File: openshift-install-powervs [hunk 1]
```diff
@@ -983,7 +983,7 @@ function variables {
   ALL_NETS=$($CLI_PATH pi nets --json| grep name | cut -f4 -d\" | grep -v "\-pub-net" || true)
   [ -z "$ALL_NETS" ] && error "No private network found"
 
-  RAW_OCP_VERSIONS=$(curl -skL https://mirror.openshift.com/pub/openshift-v4/ppc64le/clients/ocp/| grep "$RELEASE_VER" | cut -f2 -d '>' | cut -f1 -d '<' | tac)
+  RAW_OCP_VERSIONS=$(curl -skL https://mirror.openshift.com/pub/openshift-v4/ppc64le/clients/ocp/| grep "$RELEASE_VER" | cut -f2 -d '>' | cut -f1 -d '<' | tail -r)
```
Comment (sajauddin) [inline openshift-install-powervs:986]: tail do not have option "-r"

**Review by Prajyot-Parab — APPROVED** [review]
/lgtm

**Review by Prajyot-Parab — APPROVED** [review]
/lgtm

**Review by sajauddin — APPROVED** [review]
/lgtm

**Pull request merged**